### PR TITLE
add the readme comment for global hub grafana service

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,9 @@ The manager is used to persist the data into the postgreSQL. The data is from Ka
 ### Multicluster Global Hub Agent
 The agent is running in the regional hub clusters. It is responsible to sync-up the data between the global cluster hub and the regional hub clusters. For instance, sync-up the managed clusters' info from the regional hub clusters to the global hub cluster and sync-up the policy or application from the global hub cluster to the regional hub clusters.
 
+### Multicluster Global Hub Observability
+Grafana runs on the global hub cluster, as the main service for Global Hub Observability. The Postgres data collected by the Global Hub Manager services as its default DataSource. By exposing the service via route(`multicluster-global-hub-grafana`), you can access the global hub grafana dashboards just like accessing the openshift console.
```
Comment (clyang82) [inline README.md:25]: we need a separated section in `doc` folder to cover the followings:
1. how to access grafana? `oc get route`
2. how to provide the postgres-secret to contain the ca and postgresql endpoint. please note that we use grafana server mode to access the sample postgresql. for BYO postgres, we need to use client mode to access.

Comment (yanmxa) [reply README.md:25]: Yes, I have tested provisioning the Postgres database by AWS RDS and it works.

**Review by morvencao — COMMENTED** [review]
/lgtm

**Pull request merged**